src/App/ComponentsDocumentation/components/InputField/index.js

```diff
@@ -61,16 +69,23 @@ const DeveloperDocumentation = () => (
         <section>
             <h3 className="mt-3">Autocomplete</h3>
             <p>Add <CodeTags type="primary" code="autocomplete"/> attributes to input fields when it is possible to make a form easier for users to fill. </p>
+            <p>Please check out <Link to="https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete">this site</Link> for full autocomplete specs.</p>
```

Comment:
improve the wording fo rthe link to MDN, to make it more clear what is the expected result of clicking the link 😉